### PR TITLE
[+1] Added maximum number of days option

### PR DESCRIPTION
**Pull request by Yalrafih**
https://blacklinegps.atlassian.net/browse/SW-8595
https://blacklinegps.atlassian.net/browse/SW-8596

relates to: https://github.com/blackline/icarus/pull/1829

### CODE REVIEW

File: js/daterangepicker.jQuery.js [hunk 1]
```diff
@@ -378,12 +406,12 @@
                             $(this).removeClass('ui-state-hover');
                         })
                     .bind('disable', function () {
-                        $(this).attr('disabled', true);
```
Comment (rmill) [inline js/daterangepicker.jQuery.js]: Do we know if the disabled attribute/class this this object isn't used anywhere else in the code? We might want to leave this but also disable the button

Comment (Yalrafih) [inline js/daterangepicker.jQuery.js]: @rmill ```($this)``` here refers to ```$('<div class="input-block-level"><button class="btn btn-small btn-success pull-right"><i class="icon-ok"></i>'+ options.doneButtonText +'</button></div>')``` which is the div containing the button, setting the disabled attribute for that won't do anything - the intention here was to disable the button not the div.

**Comment by rmill** [comment]
:+1:

**Comment by bdgregor** [comment]
:+1:

**Pull request merged**